src/ts/uifactory.ts

```diff
@@ -308,7 +308,7 @@ export namespace UIFactory {
       components: [
         new BufferingOverlay(),
         new AdClickOverlay(),
-        new PlaybackToggleOverlay(),
+        new PlaybackToggleOverlay({ enterFullscreenOnInitialPlayback: false }),
```

Comment:
this is duplicated a couple of times... 

can we extract this into a function to be reusable?